astro/src/content/docs/get-started/core-concepts/_template-settings.mdx

```diff
@@ -0,0 +1,76 @@
+import APIBlock from 'src/components/api/APIBlock.astro';
+import APIField from 'src/components/api/APIField.astro';
+import InlineField from 'src/components/InlineField.astro';
+import EnterprisePlanBlurbApi from 'src/content/docs/_shared/_enterprise-plan-blurb-api.astro';
+
+<APIBlock>
+  <APIField name="Forgot password" optional>
+    The email or message template to use for the forgot password workflow.
+  </APIField>
+  <APIField name="Identity update" optional since="1.30.0">
```

Comment:
Versioning is weird for some of these. This one was renamed in the UI and added phone template for `1.99.9`, but it has existed in some form since `1.30.0`.